src/components/users/UsersIndex.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component, useReducer } from 'react';
+import Table from '../Table';
+
+// library for AJAX functions
+const axios = require('axios');
```

Comment:
`import axios from 'axios';`

Reply:
I'll abstract our API into a library - that way we can add authentication tokens

Reply:
Implemented change. Okay cool that will help alot.